_config/version.yml

```diff
@@ -5,4 +5,6 @@ After: 'framework/*'
 SilverStripe\Core\Manifest\VersionProvider:
   modules:
     silverstripe/framework: ''
+    silverstripe/recipe-core: Core Recipe
```

Comment:
Would it make more sense for this to be preferred over silverstripe/cms?  The recipe version paints a broader picture of what you're "running", though the version of the CMS module could vary 🤔

Reply:
Order based of revision Max recommended https://github.com/silverstripe/silverstripe-framework/pull/9628#issuecomment-669629976

Also recipe-core lacks CMS, so it probably makes no difference if we swapped them.

Reply:
Okay, in that case maybe the `recipe-core` reference belongs in framework, since it'll never be used here 🤔

Reply:
Framework doesn't have this config cos the feature lives in CMS.

No much point having a CMS version indicator if you have a CMSless install =)

Reply:
CMS brings SiteTree, mainly, but you can still have an admin back-end and manage things through model admins e.g. for an API provider sort of installation. Does silverstripe-admin have this feature to report on framework it's using?

Reply:
Had a bit of a dig, and it appears this functionality lives in the _admin_ module, not the cms module. I wasn't suggesting moving the CMS entry to framework, just the `recipe-core` one, since it will always be overridden by the CMS entry if it's listed here anyway.

My final recommendation: Remove the `recipe-core` reference here since it's inert. If we want it to actually be a viable choice for the `VersionProvider`, add it back either in framework or admin.

Reply:
@Cheddam have removed the mention of recipe-core.  We probably not need a PR in framework or admin since VersionProvider is only ever used within the CMS.  We probably should never even mentioned it in the ACs

Reply:
I think you're conflating "the CMS" with the Admin UI. Here's a project that uses `recipe-core` and `admin` - the `cms` module is not present:

![Screen Shot 2020-08-26 at 14 15 57](https://user-images.githubusercontent.com/242621/91247576-b249f480-e7a6-11ea-91d3-6114b1c8af92.png)

So I think another PR to framework or admin makes sense. Regardless, the part pertaining to this PR is resolved 👍